[OKTA-543635] Reserved attributes in SCIM (#4937)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md

**Q: How much support is required for filtering results?**

> The filtering capabilities in the SCIM protocol are broad, but the common filtering use case with Okta is narrow. Okta determines if a newly created user or group exists in your app based on a matching identifier. This means the `eq` (equals) operator is all you really need to support.

Reviewer: ```suggestion
The filtering capabilities in the SCIM protocol are broad, but the common filtering use case with Okta is narrow. Okta determines if a newly created user or group exists in your app based on a matching identifier. This means the `eq` (equals) operator is all you need to support.
```

**Q: What is the timeout if Okta doesn't receive a response from my SCIM server?**

The timeout is 60 seconds if Okta doesn't receive anything from the socket after any SCIM request is made.

**Q: Are there reserved attributes that I can't use?**

> Okta follows the SCIM specifications as defined by [RFC 7642](https://datatracker.ietf.org/doc/html/rfc7642), [RFC 7643](https://datatracker.ietf.org/doc/html/rfc7643), and [RFC 7644](https://datatracker.ietf.org/doc/html/rfc7644), with exceptions listed in the reference guides. All system or reserved attributes follow these RFC documents. For example, `groups` is a reserved and used property on the `User` object defined as part of RFC 7643, so you shouldn't try to create a custom attribute called `groups` in your SCIM server.

Reviewer: ```suggestion
Okta follows the SCIM specifications as defined by [RFC 7642](https://datatracker.ietf.org/doc/html/rfc7642), [RFC 7643](https://datatracker.ietf.org/doc/html/rfc7643), and [RFC 7644](https://datatracker.ietf.org/doc/html/rfc7644), with exceptions listed in the reference guides. All system or reserved attributes follow these RFC documents. For example, `groups` is a reserved and used property on the `User` object defined as part of RFC 7643. Therefore, don't create a custom attribute called `groups` in your SCIM server.
```
